SConstruct

```diff
@@ -1,19 +1,26 @@
 # -*- python -*-
 
+# Importing "lsst.sconsUtils" will cause Python
+# to load the __init__.py which will cause lsstimport to
+# be loaded which will hopefully not fail with a fatal error
+# but will initially print out a warning that base has not
+# been built yet.
```

Comment:
What I'm trying to say is that `lsstimport` should be designed such that attempting load it before `lsst64defs` is created should not be fatal. A lot of this is irrelevant if we abandon `lsst64defs`.